src/FlipMove.js

```diff
@@ -574,15 +583,30 @@ class FlipMove extends Component<ConvertedProps, FlipMoveState> {
       // If the child element returns null, we need to avoid trying to
       // account for it
       if (!childData.domNode || !child) {
+        childrenBoundingBoxes.push(null);
         return;
       }
 
-      this.setChildData(childKey, {
-        boundingBox: getRelativeBoundingBox({
+      childrenBoundingBoxes.push(
+        getRelativeBoundingBox({
           childDomNode: childData.domNode,
           parentDomNode,
           getPosition: this.props.getPosition,
         }),
+      );
+    });
+
+    this.state.children.forEach((child, index) => {
+      const childKey = getKey(child);
+
+      const childBoundingBox = childrenBoundingBoxes[index];
```

Comment:
Thanks for the variable name!

Could you quickly explain why it isn't being used in the early-return conditional anymore? I assume it's because `null` is a valid type for `NodeData.boundingBox`, and so if that's what the value is currently resolved as, it makes sense to set it to that? This makes sense to me, just wanna make sure this was an intentional omission and I'm not missing anything :)

Gonna accept the PR since it's not-blocking but if you happen to see this, please let me know!

Reply:
It's just a small optimization because `childBoundingBox` would be `null` if `childKey` is falsey value (see line 568) and `childrenBoundingBoxes` have the same order as `children`.